### DISCUSSION TITLE
Error htmlspecialchars(): Argument #1 ($string) must be of type string, array given

### DISCUSSION THREAD

**Discussion by cabelq** [Power-Components/livewire-powergrid#1331] [General]
I installed PowerGrid according to the official documentation. But when I want to use the livewire component it gives me the error
htmlspecialchars(): Argument #1 ($string) must be of type string, array given

![errorpg](https://github.com/Power-Components/livewire-powergrid/assets/12152688/58c971c5-6716-4578-b009-63052e11026f)

What am I doing wrong?
Thank you

**Comment by marineusde** [comment]
In your screenshot, I see it was Laravel 10.40, right? There was a "buggy feature", its solved in 10.41, so please upgrade :)
